app/views/api/v2/hosts/main.json.rabl

```diff
@@ -10,7 +10,7 @@ extends "api/v2/hosts/base"
 attributes :ip, :environment_id, :environment_name, :last_report, :mac, :realm_id, :realm_name,
            :sp_mac, :sp_ip, :sp_name, :domain_id, :domain_name, :architecture_id, :architecture_name, :operatingsystem_id, :operatingsystem_name,
            :subnet_id, :subnet_name, :sp_subnet_id, :ptable_id, :ptable_name, :medium_id, :medium_name, :build,
-           :comment, :disk, :installed_at, :model_id, :hostgroup_id, :hostgroup_name, :owner_id, :owner_type,
```

Comment:
It's on the verge of breaking backwards compatibility IMO, but still fine. @ohadlevy this is good to merge if you ack it